src/controllers/categories.js

```diff
@@ -266,7 +266,13 @@ categoriesController.get = function(req, res, next) {
 			res.locals.linkTags.push(rel);
 		});
 
-		res.render('category', data);
+		plugins.fireHook('filter:category.build', {req: req, res: res, templateData: data}, function(err, data) {
+			if (err && process.env === 'development') {
```

Comment:
Can you change this and the other one in topics.js to 
```
if (err) {
    return next(err);
}
```
The error handler in routes/index.js will print errors.